www/json/trip_confirm_options.json.sample

```diff
@@ -1,25 +1,41 @@
 {
+
   "modeOptions" : [
-       {"text":"Walk", "value":"walk"},
```

Comment:
@deepalics0044 this is a sample file (`trip_confirm_options.json.sample`). In general, instead of editing the sample file directly (which can cause merge conflicts later), you want to copy it into (`trip_confirm_options.json`). Can you make that change?